led_hook_udp_node:
  ros__parameters:
    udp_targets:
      - "10.1.100.210:8000"
      - "10.1.100.211:8001"
    send_ascii: false   # ส่งเป็นไบต์เดียว int8
